test/specs/RangeHighlightsSpec.js

```diff
@@ -38,6 +40,39 @@ describe("RangeHighlights Render Tests", function() {
                 { "start": 23, "end": 15 }  //out of range - not visible
             ]
         });
+
+        testSlider3 = $('#testSlider3').slider({
+            id: 'slider3',
+            min: 0,
+            max: 20,
+            step: 1,
+            value: 14,
+            reversed: true,
+            rangeHighlights: [
+                { "start": 2, "end": 5 },    // left: 75%; width: 15%
+                { "start": 7, "end": 8 },    // left: 60%; width: 5%
+                { "start": 17, "end": 19 },  // left: 5%; width: 10%
+                { "start": 17, "end": 24 },  //out of range - not visible
+                { "start": -3, "end": 19 }   //out of range - not visible
+            ]
+        });
+
+        testSlider4 = $('#testSlider4').slider({
```

Comment:
could you properly tear down `testSlider4` in the `afterEach` block (lines 79 - 85)?